webdriver/conftest.py

```diff
@@ -36,6 +39,25 @@ def session(_session, request):
 
     return _session
 
+
 @pytest.fixture(scope="function")
 def http(session):
     return HTTPRequest(session.transport.host, session.transport.port)
+
+
+@pytest.fixture
+def server_config():
+    return json.loads(os.environ.get("WD_SERVER_CONFIG"))
```

Comment:
Still, is there a suitable default value? To run this locally, what would I set in WD_SERVER_CONFIG?